migrate/scripts/gauge-name.js

```diff
@@ -0,0 +1,63 @@
+const qs = require('querystring');
+
+const urltoGaugeName = url => {
+  return qs.decode(url).title.toLowerCase();
```

Comment:
So I would like this not to have `toLowerCase()` and instead we lowercase it when we retrieve it out of the database for comparison.